### DISCUSSION TITLE
Changing the format for supporting email id

### DISCUSSION THREAD

**Discussion by Jeevan-23** [community/community#62996] [GitHub Education]
I'm from GM INSTITUTE OF TECHNOLOGY our college gives student I'd as @gmit.ac.in but in GitHub the supported id is on.microsoft.com so I'm unable to apply for student developer pack and campus expert 
So please look into this 
@github

**Comment by fabricio-fn** [comment]
if college allows try using @gmail.com

**Comment by mehulrajdev** [comment]
Change in the setting of GitHub. @yahoomail.com @gmail.com
